analysis/analysis_utils.py

```diff
@@ -194,14 +193,14 @@ def load_images(self, im_filepath, time_file, psf_val=1.4, mjd_lims=None):
         stack = kb.image_stack(images)
 
         # Apply masks
-        stack.apply_mask_flags(flags, flag_exceptions)
-        stack.apply_master_mask(master_flags, 2)
+        #stack.apply_mask_flags(flags, flag_exceptions)
+        #stack.apply_master_mask(master_flags, 2)
 
-        stack.grow_mask()
-        stack.grow_mask()
+        #stack.grow_mask()
+        #stack.grow_mask()
         
         # This applies a mask to pixels with more than 120 counts
-        #stack.apply_mask_threshold(120.)
+        stack.apply_mask_threshold(120.)
```

Comment:
Should this be turned back on?

Reply:
I think so. Without using bitmasks, having a pixel threshold eliminates a lot of the ultra-high likelihood objects (e.g. 100+).